.travis.yml

```diff
@@ -9,11 +9,14 @@ python:
 env:
   - DJANGO="django>=1.11,<1.12"
   - DJANGO="django>=2.0,<2.2"
+  - DJANGO="django>=2.2,<2.3"
```

Comment:
replace `2.3` with `3.0` since in this case the next django version after 2.2 is 3.0.

Reply:
Done :smile: